pymchelper/readers/shieldhit/reader_bdo2016.py

```diff
@@ -15,6 +15,7 @@
 class SHReaderBDO2016(SHReader):
     """
     Binary format reader from version >= 0.6
```

Comment:
but only until 0.8.0

Reply:
0.6.0 <= version < 0.8.0

Reply:
Not really, if old `detect.dat` format is used, then we still end up with bdo2016 file format even in SH12A v0.9.0